.storybook/main.js

```diff
@@ -13,7 +12,11 @@ module.exports = {
     config.plugins.push(new NodePolyfillPlugin());
     return config;
   },
-  core: {
-    builder: 'webpack5',
+  framework: {
+    name: '@storybook/nextjs',
+    options: {},
+  },
+  docs: {
+    autodocs: true,
```

Comment:
These new docs pages are not working great in all cases:

![image](https://github.com/opencollective/opencollective-frontend/assets/1556356/22336ebe-f2e8-4007-8444-a8e056b8d7db)

Reply:
To load images properly: https://github.com/opencollective/opencollective-frontend/pull/8904/files#diff-cafe2123a72c4ce3a9f7e9ee4b0e188256eb02d7ec1e54fa2cadfac99b22f92bR13